test/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/example/example.dart

```diff
@@ -0,0 +1,20 @@
+// ignore_for_file: unused_element
+const _defaultRadius = BorderRadius.all(16);
+final _defaultFinalRadius = BorderRadius.circular(8); // LINT
```

Comment:
Please add more test cases for the wrong usage: for instance, border radius as a class static field, as a filed declared inside build method and as inlined `BorderRadius.circular(8)` into a widget template.